builtin-features/reservedWords.inc

```diff
@@ -6,24 +6,24 @@ packToken trueToken = packToken(true);
 packToken falseToken = packToken(false);
 packToken noneToken = packToken::None();
 
-void True(const char* expr, const char** rest, rpnBuilder* data) {
+void True(const char*, const char**, rpnBuilder* data) {
   data->handle_token(trueToken->clone());
 }
 
-void False(const char* expr, const char** rest, rpnBuilder* data) {
```

Comment:
This isn't a big issue, but it is often preferable to avoid making style changes on PRs, for 3 reasons:

1. It might differ from the code style on the rest of the code and it is always better to have a single idiom for everything on the same code-base.
2. This might cause conflicts with the linter (it didn't in this case, so I won't ask you to change, it is ok)
3. It forces who is reviewing your code to read several lines where no real changes were made.

Reply:
I am ok with not changing the style and strived to stuck with the genuine one for the reasons above.   The motivation behind the removal of the useless 'expr' is to get rid of a compilation warning. I tend to fix these warnings as some innocuous ones may hide lethal issues.

Reply:
No problem, this one isn't causing problems with the linter, so it's alright.

Reply:
I will mark this one as resolved as well.